packages/size-limit/get-config.js

```diff
@@ -91,6 +96,18 @@ function toName(files, cwd) {
  */
 const dynamicImport = async filePath => (await import(filePath)).default
 
+/**
+ * Loads a TypeScript file from a given file path using the
+ * {@linkcode jiti} function. This loader function simplifies the
+ * process of dynamically importing TypeScript modules at runtime,
+ * offering a way to execute or import TypeScript files directly
+ * without pre-compilation.
+ *
+ * @param {string} filePath - The path to the TypeScript file to be loaded.
+ * @returns {any} The module exports from the loaded TypeScript file.
+ */
+const tsLoader = filePath => jiti(filePath)
```

Comment:
Should we move `jiti` creating here to avoid unnecessary init for non-TS configs?

Reply:
That's not a bad idea, I can use a dynamic import to import `jiti`.